lib/node_modules/@stdlib/math/base/special/uimuldw/examples/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var uimuldw = require( './../lib' );
+
+var i;
+var j;
+var y;
+
+for ( i = 0xFFFFFFF0; i < 0xFFFFFFFF; i++ ) {
+	for ( j = i; j < 0xFFFFFFFF; j++) {
+		y = uimuldw( i, j );
+		console.log( '%d x %d = 0x%s%s', i, j, y[0].toString( 16 ).padStart( 8, 0 ), y[1].toString( 16 ).padStart( 8, 0 ) );
```

Comment:
`padStart` is not supported in older environments. Would you mind updating to use [`@stdlib/string/left-pad`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/string/left-pad)? Thanks!